modtree/modtree
# fix: edges are drawn from newly placed nodes

### Summary of changes

- Fixes #280. Merge after #295
- Change dagre ranker to `network-simplex`

### Testing

- Remove CS1010 and CS2030 if they are in your graph
- Try adding CS1010 and CS2030. They should be joined by an edge
